### PR TITLE
Docs: Part 5

### PR DESCRIPTION
**Pull request by raulgomezc**
Closes [Ticket](https://trello.com)

## Testing

Steps to manually verify the change:
1. _Setup_: step to set the environment
2. _Exercise_: How to run the environment and example link: [Test Link](http://127.0.0.1:8000/)
3. _Verify_: what we need to verify

### CODE REVIEW

File: docs/tutorial/part05.rst [hunk 1]
```diff
@@ -2,5 +2,118 @@
 How to create the API.
 ======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Why the API structure
+=====================
+
+We decided to use APIs to fetch the data to make queries more effective, as sometimes queries get too big, we decided it was better to fetch the data in little chunks rather than one big request and that's what APIs allows us to do.
```
Comment (mariamera) [inline docs/tutorial/part05.rst]: maybe a dot instead of comma in `get too big, we decided it was better to`

File: docs/tutorial/part05.rst [hunk 2]
```diff
@@ -2,5 +2,118 @@
...
\ No newline at end of file
+Why the API structure
+=====================
+
+We decided to use APIs to fetch the data to make queries more effective, as sometimes queries get too big, we decided it was better to fetch the data in little chunks rather than one big request and that's what APIs allows us to do.
+
+Explain the library
+===================
+
+In this project, we are using the rest_framework_ library because it has some simple non-limiting ways to implement an API structure, if you desire you can use any of your choosing, but the examples presented ahead will be based on the behavior of this specific library.
```
Comment (mariamera) [inline docs/tutorial/part05.rst]: same here in ` implement an API structure, if you desire `

File: docs/tutorial/part05.rst [hunk 3]
```diff
@@ -2,5 +2,118 @@
...
+    from django.conf.urls import url
+    from .views import GetNodesData
+
+    urlpatterns = [
+        url(r’^nodes[/]?$’, GetNodesData.as_view(), name=‘get_nodes_data’),
+    ]
+
+Now on you can hit the endpoint with a method of your choosing, for this example, we are going to use curl, on your console:
+
+``curl http://127.0.0.1:8000/fetch/nodes``
```
Comment (mariamera) [inline docs/tutorial/part05.rst]: Nice example 👍 💛

File: docs/tutorial/part05.rst [hunk 4]
```diff
@@ -2,5 +2,118 @@
...
+
+You should see:
+
+``"Temporary Data"``
+
+The other 5 endpoints with its classes/URLs respectively are :
```
Comment (oscarpolanco) [inline docs/tutorial/part05.rst]: Extra space before the dots

File: docs/tutorial/part05.rst [hunk 5]
```diff
@@ -2,5 +2,118 @@
...
+In this project, we are using the rest_framework_ library because it has some simple non-limiting ways to implement an API structure, if you desire you can use any of your choosing, but the examples presented ahead will be based on the behavior of this specific library.
+
+    .. _rest_framework: http://www.django-rest-framework.org/
+
+Install the library
+===================
+
+Just run::
+
+    pip install rest_framework
```
Comment (oscarpolanco) [inline docs/tutorial/part05.rst]: This is already in the requirement file; maybe the user already has this if he is following the tutorial. Maybe a note pointing this should be useful?
What do you think? 🤔

**Review by oscarpolanco — APPROVED** [review]
LGTM

File: docs/tutorial/part05.rst [hunk 6]
```diff
@@ -2,5 +2,126 @@
...
+In this project, we are using the rest_framework_ library because it has some simple non-limiting
+ways to implement an API structure. If you desire you can use any of your choosing, but the examples
+presented ahead will be based on the behavior of this specific library.
+
+    .. _rest_framework: http://www.django-rest-framework.org/
+
+You should have the library already installed since it is in the requirements file that we provided
+for the project. If not just run::
+
+    pip install rest_framework
```
Comment (cyuli) [inline docs/tutorial/part05.rst]: No matching distribuition found for rest_framework. Maybe it refers to djangorestframework library. No?

File: docs/tutorial/part05.rst [hunk 7]
```diff
@@ -2,5 +2,126 @@
...
+
+    from rest_framework.views import APIView
+    from rest_framework.response import Response
+    from rest_framework import status
+
+    class GetNodesData(APIView):
+        def get(self, request):
+            return Response(‘Temporary Data’, status=status.HTTP_200_OK)
+
+Now on your ``fetch_api/urls.py`` add the endpoint to the urls::
```
Comment (cyuli) [inline docs/tutorial/part05.rst]: It doesn't exists, i think that would be good to say that they should to create it.

File: docs/tutorial/part05.rst [hunk 8]
```diff
@@ -2,5 +2,126 @@
 How to create the API.
 ======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Why the API structure
```
Comment (EnmanuelParache) [inline docs/tutorial/part05.rst]: Question (?) mark?

File: docs/tutorial/part05.rst [hunk 9]
```diff
@@ -2,5 +2,126 @@
...
+=====================
+
+We decided to use APIs to fetch the data to make queries more effective, as sometimes queries get
+too big. We decided it was better to fetch the data in little chunks rather than one big request.
+The API structure allows us to do so and to have a single page application. We will be returning
+JSON to the calls since it is easily usable on the frotend side.
+
+Explain the library
```
Comment (EnmanuelParache) [inline docs/tutorial/part05.rst]: 'Explain the library' sounds weird (boring to be honest). Instead use something like 'REST API' or 'Implementing REST framework for API' it's more attractive and useful for somebody that is only looking for this.

File: docs/tutorial/part05.rst [hunk 10]
```diff
@@ -2,5 +2,126 @@
...
+Now create this file ``fetch_api/urls.py`` and fill it with this::
+
+    from django.conf.urls import url
+    from .views import GetNodesData
+
+    urlpatterns = [
+        url(r'^nodes[/]?$', GetNodesData.as_view(), name='get_nodes_data'),
+    ]
+
+Now you can hit the endpoint with a method of your choosing, for this example, we are going to use
```
Comment (EnmanuelParache) [inline docs/tutorial/part05.rst]: method of your preference

File: docs/tutorial/part05.rst [hunk 11]
```diff
@@ -2,5 +2,126 @@
...
+
+    pip install djangorestframework
+
+On your console and restart your Django server.
+
+Create the views / endpoints
+============================
+
+We are going to need a couple different endpoints to make our application work, but for now, we will
+focus on one to explain the process of making them, we are going to be building the
```
Comment (mariamera) [inline docs/tutorial/part05.rst:33]: I think it is better a dot instead of a comma after 'them' and before 'we'

File: docs/tutorial/part05.rst [hunk 12]
```diff
@@ -2,5 +2,126 @@
...
+
+    from rest_framework.views import APIView
+    from rest_framework.response import Response
+    from rest_framework import status
+
+    class GetNodesData(APIView):
+        def get(self, request):
+            return Response('Temporary Data', status=status.HTTP_200_OK)
+
+Now create this file ``fetch_api/urls.py`` and fill it with this::
```
Comment (mariamera) [inline docs/tutorial/part05.rst]: the file ``fetch_api/urls.py`` is created in part 3.

 You can see it here: https://github.com/neo4j-examples/paradise-papers-django/pull/51/files#diff-c1e0a5d898826045a3e281cad742e750R25

**Comment by mariamera** [comment]
The file `nodeutils.py` is never created on the tutorial , I think we should add this here.

**Review by mariamera — CHANGES_REQUESTED** [review]
.

**Pull request merged**